src/Resources/contao/dca/tl_layout.php

```diff
@@ -272,8 +272,17 @@
 			'inputType'               => 'select',
 			'options'                 => array('external_first', 'internal_first'),
 			'reference'               => &$GLOBALS['TL_LANG']['tl_layout'],
+			'eval'                    => array('tl_class'=>'w50'),
 			'sql'                     => "varchar(16) NOT NULL default ''"
 		),
+		'combineScripts' => array
+		(
+			'label'                   => &$GLOBALS['TL_LANG']['tl_layout']['combineScripts'],
+			'exclude'                 => true,
+			'inputType'               => 'checkbox',
+			'eval'                    => array('tl_class'=>'w50 m12'),
+			'sql'                     => "char(1) NOT NULL default ''"
```

Comment:
I wonder if the default must be `1`.
As previous versions would combine everytime, it is a change in default behaviour when upgrading.

IMO we must at least document and communicate this prominently when it defaults to off.